plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/protobuf/TestKafkaProtobufWithSchemaRegistryMinimalFunctionality.java

```diff
@@ -171,10 +171,10 @@ public void testUnsupportedRecursiveDataTypes()
         waitUntilTableExists(topic);
         assertQueryFails("SELECT * FROM " + toDoubleQuoted(topic),
                 "Protobuf schema containing fields with self-reference are not supported because they cannot be mapped to a Trino type: " +
-                        "io.trino.protobuf.RecursiveValue > " +
-                        "io.trino.protobuf.RecursiveStruct > " +
-                        "io.trino.protobuf.RecursiveStruct.FieldsEntry > " +
-                        "io.trino.protobuf.RecursiveValue");
+                        "io.trino.protobuf.schema.recursive_value_one: io.trino.protobuf.RecursiveValue > " +
+                        "io.trino.protobuf.RecursiveValue.struct_value: io.trino.protobuf.RecursiveStruct > " +
+                        "io.trino.protobuf.RecursiveStruct.fields: io.trino.protobuf.RecursiveStruct.FieldsEntry > " +
+                        "io.trino.protobuf.RecursiveStruct.FieldsEntry.value: io.trino.protobuf.RecursiveValue");
```

Comment:
This error message goes to user, they are not much aware of trino java packages. Should use field names from protobuf schema?

Reply:
The field names are from the protobuf schema, see https://github.com/trinodb/trino/pull/15724/files#diff-8ca538d4195acbb011666dca6daee504059c8090ffbedda1cf1fb8389c1e4827R3.